downstream/modules/eda/con-modifying-simultaneous-activations.adoc

```diff
@@ -0,0 +1,17 @@
+[id="modifying-simultaneous-activations"]
+
+= Modifying the number of simultaneous rulebook activations 
+
+[role="_abstract"]
+By default, {EDAcontroller} allows 12 rulebook activations to run simultaneously.
+If any more than 12 rulebook activations are created, the expected behavior is that subsequent rulebook activations wait in pending until there is an available rulebook activation worker.
+In this case, the rulebook activation status will display as “pending” even if there is enough free memory and CPU on your {EDAcontroller} instance.
+To change this behavior, you must change the default maximum number of running rulebook activations.
+
+**Note:** The value for `EDA_MAX_RUNNING_ACTIVATIONS` doesn’t change with the change in instance size, so it needs to be adjusted manually.
```

Comment:
I would follow the note syntax shown [here](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions)--this is how I see the majority of our notes formatted.

Reply:
Will apply this change to the content when I update the other sections that are changing for 2.5.